OpenRA.Platforms.Default/OpenAlSoundEngine.cs

```diff
@@ -284,10 +313,30 @@ public void PauseSound(ISound sound, bool paused)
 			var source = ((OpenAlSound)sound).Source;
 			int state;
```

Comment:
Would it be better to copy the lines 314-339, paste it into a private method (something like `pause(Source source, bool paused)`) and let `PauseSound` and `SetAllSoundsPaused` use it, as both methods use almost the same code?

Reply:
It would be better. Changed.